src/local_scheduler/lib/java/org_ray_spi_impl_DefaultLocalSchedulerClient.cc

```diff
@@ -208,6 +208,33 @@ Java_org_ray_spi_impl_DefaultLocalSchedulerClient__1reconstruct_1object(
   local_scheduler_reconstruct_objects(client, {*o.PID}, fetch_only);
 }
 
+/*
+ * Class:     org_ray_spi_impl_DefaultLocalSchedulerClient
+ * Method:    _reconstruct_objects
+ * Signature: (J[B)V
+ */
+JNIEXPORT void JNICALL
+Java_org_ray_spi_impl_DefaultLocalSchedulerClient__1reconstruct_1objects(
```

Comment:
Let's only keep this plural version in c++ code, and have the java single version call the java plural version.